### PR TITLE
Remove trivyignore

### PR DESCRIPTION
**Pull request by arturo-seijas**
Removed from the trivyignore file two issues already fixed in pebble and added a new detected vulnerabilities

### CODE REVIEW

**Review by jdkandersson — DISMISSED** [review]

File: .trivyignore [hunk 1]
```diff
@@ -5,13 +5,12 @@ CVE-2020-29652
...
 CVE-2021-43565
 CVE-2021-44716
-CVE-2022-27191
 CVE-2022-27664
 CVE-2022-28948
 CVE-2022-3064
 CVE-2022-32149
 CVE-2022-41723
 CVE-2023-2515
+CVE-2021-38561
```
Comment (yanksyoon) [inline .trivyignore:16]: I think there's a final new line missing :D

**Review by yanksyoon — DISMISSED** [review]
LGTM, minor comment!

**Review by merkata — DISMISSED** [review]
Looks good, can you update the description before merging ? Two CVEs are removed and one added so would be good to know why/how. Thanks!

**Review by yanksyoon — DISMISSED** [review]

**Review by yanksyoon — APPROVED** [review]

**Comment by amandahla** [comment]
It's me or they are the same?

**Comment by arturo-seijas** [comment]
> It's me or they are the same?

Ok! They are

**Pull request closed without merging**